recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "1.12.1" %}
-{% set sha256 = "031c71073db73da732b5d01710220564ce6dd88d812ba053f0cc94296401eccb" %}
+{% set version = "1.13.1" %}
+{% set sha256 = "dbc229ee9750b02b514937d017744443a269ea0241ed3f32b9af0703589d25d4" %}
 
 package:
   name: pytorch
   version: {{ version }}
 
 source:
-  # for local testing use a tarball including submodules
+  # for local testing use a tarball including submodules. The "pytorch-v" tarballs contain submodules; the "pytorch-"
```

Comment:
Do we want to be using the tarball including submodules for release?

Reply:
good question. Probably? If we use the version that includes the submodules, there's less chance that one of the submodule projects could re-release a version over the top and break something, or remove the project from where it was, etc